webpack.config.js

```diff
@@ -69,6 +70,9 @@ module.exports = {
   devtool: isProduction ? 'source-map' : 'cheap-module-eval-source-map',
   plugins: (() => {
     let plugins = [
+      new FlowWebpackPlugin({
+        failOnError: false
```

Comment:
This was supposed to be the default, but it wasn't behaving that way. Now it's explicit. There's another config called `failOnErrorWatch` that is also supposed to be `false` by default. I'm not sure if one is better, but this seems to be working with our `npm start`.

Reply:
If we do include this plugin, we should also change the reportingSeverity to warning.  This would allow errors showing up in the console to be more quickly disregarded during a normal dev workflow.

```js
new FlowWebpackPlugin({
  failOnError: false,
  reportingSeverity: 'warning'
}),